config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "home#index"
+  get '/users/:id', to: 'users#show'
+  get '/users/:id/posts', to: 'posts#index'
+  get '/users/:user_id/posts/:id', to: 'posts#show'
```

Comment:
- [ ] @jicamargo Great work on your routes however, they don't match the URLs in the project description check the comparison below: the user's routes is missing the posts routes for the index action.

# your routes
![image](https://github.com/jicamargo/blogify/assets/61408860/a2314e1c-5b68-4cca-98c8-3969a6d07a78)

# project routes
![image](https://github.com/jicamargo/blogify/assets/61408860/e613faf5-5dbb-4b8a-8c02-f965a577f5b9)